plugins/woocommerce-admin/client/index.js

```diff
@@ -140,7 +150,23 @@ if (
 				)
 			).render( <SettingsPaymentsMainWrapper /> );
 		}
-		// Todo: SettingsPaymentsOfflineWrapper
-		// Todo: SlotFill for WooPayments & others
+
+		if ( paymentsOfflineRoot ) {
+			createRoot(
+				paymentsOfflineRoot.insertBefore(
+					document.createElement( 'div' ),
+					null
+				)
+			).render( <SettingsPaymentsOfflineWrapper /> );
+		}
+
+		if ( paymentsWooCommercePaymentsRoot ) {
+			createRoot(
+				paymentsWooCommercePaymentsRoot.insertBefore(
+					document.createElement( 'div' ),
+					null
+				)
+			).render( <SettingsPaymentsWooCommercePaymentsWrapper /> );
+		}
```

Comment:
We've been keeping slot rendering for for Settings pages in one place

https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/settings/settings-slots.js

It would be good to move this logic so its all in one place.

Reply:
@psealock For main, offline payments, and WooPayments we did not use SlotFill due since they're not intended to be extended, and the components live in core